### PR TITLE
remove C2R move to convert role

### PR DESCRIPTION
**Pull request by solumath**
I input the `remove of convert 2 rhel` to wrong role. Should be in convert role.

### CODE REVIEW

File: roles/convert/tasks/main.yml [hunk 1]
```diff
@@ -6,4 +6,7 @@
     reboot_timeout: 120
   timeout: 180
   when: convert_reboot_requested
+
+- name: Remove Convert2RHEL
+  ansible.builtin.include_tasks: remove-c2r.yml
```
Comment (solumath) [inline roles/convert/tasks/main.yml:44]: I was thinking maybe we shouldn't remove it if the conversion was unsuccessful. If so then it should be changed also here https://github.com/oamg/infra.convert2rhel/pull/9.
What do you think?

Comment (andywaltlova) [reply roles/convert/tasks/main.yml:44]: I think it's fine to remove it if unsuccessful because the common role will install it anyway when run again.

Comment (r0x0d) [reply roles/convert/tasks/main.yml:44]: For now, I think it's totally fine to remove it every time. We could discuss this later and remove only when the conversion is successful to save some time and network, but right now, I believe this doesn't matter too much, and we can safely remove whenever we want.

**Review by r0x0d — APPROVED** [review]
looks lovely, thanks, @solumath!

**Comment by r0x0d** [comment]
Hi, @solumath! Can you rebase the PR? I think it is mostly complete and we can merge it after the rebase.

**Comment by solumath** [comment]
@r0x0d done

**Review by r0x0d — APPROVED** [review]

**Comment by r0x0d** [comment]
@solumath another rebased is required :(

**Pull request merged**